worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/io/SchematicLoadException.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * WorldEdit, a Minecraft world manipulation toolkit
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldEdit team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldedit.extent.clipboard.io;
+
+import com.sk89q.worldedit.util.formatting.text.Component;
+
+/**
+ * Raised when a known exception occurs during schematic load.
+ */
+public class SchematicLoadException extends RuntimeException {
```

Comment:
Can we mark this as non-API, i.e. with a Javadoc comment? Or maybe move it to some internal package?

Reply:
Is there a reason to do that? Given adding schematic formats is API, I can see throwing this exception as pretty useful from an API perspective

Reply:
Hmm, I guess it's fine. Maybe we can just mark it `final` then? I don't know if I want to open it up for extending that we might need to consider later.